### PR TITLE
Improve readability of parsing PITCH_BEND and POLYPHONIC_KEY_PRESSURE messages

### PR DESCRIPTION
**Pull request by jacobtylerwalls**
Noticed while debugging that the `else` case was capturing more than NOTE_ON and NOTE_OFF. So I looked up what was being captured and wrote cases for them. Needs a test file; none of the .midi files in the repo have PITCH_BEND messages, for instance.

### CODE REVIEW

File: music21/midi/__init__.py [hunk 1]
```diff
@@ -829,12 +829,20 @@ def parseChannelVoiceMessage(self, midiBytes: bytes) -> bytes:
...
+            self.parameter2 = byte2  # most significant byte
+            return midiBytes[3:]
+        elif self.type in (ChannelVoiceMessages.NOTE_ON, ChannelVoiceMessages.NOTE_OFF):
             # next two bytes:  pitch, velocity
             self.pitch = byte1
             self.velocity = byte2
             return midiBytes[3:]
+        elif self.type == ChannelVoiceMessages.POLYPHONIC_KEY_PRESSURE:
+            self.pitch = byte1
+            self.data = byte2
```
Comment (mscuthbert) [inline music21/midi/__init__.py:843]: Should this be parameter1, parameter2?

Comment (jacobtylerwalls) [reply music21/midi/__init__.py:843]: The values are pitch and pressure. Up to you.

Comment (jacobtylerwalls) [reply music21/midi/__init__.py:843]: The monophonic pressure message uses `self.data` for pressure, so that's what I was thinking.

**Comment by mscuthbert** [comment]
It is possible to generate a MIDI file with PITCH_BEND messages just by writing from music21 a score with quarter tones or other microtones -- that would at least give some data for a test.  THANKS!

File: music21/midi/__init__.py (hunk 1 above)
Comment (jacobtylerwalls) [reply music21/midi/__init__.py:843]: Looking more closely, .pitch and .data are both just synonyms for parameter 1 on setting; on getting, .pitch will only return the value if NOTE_ON or NOTE_OFF. so we definitely want parameter 1 and parameter2 because we'll lose byte2. Now that I see this I see that this patch is mostly just about readability of the code.

**Comment by jacobtylerwalls** [comment]
Coveralls observed that the PITCH_BEND case was already tested by the doctest for MidiFile:
https://github.com/cuthbertLab/music21/blob/506b28c109360d3054604dff77468da184ddcb86/music21/midi/__init__.py#L1235-L1239
